packages/web/src/app/components/memberApprovalRequiredToggle.tsx

```diff
@@ -45,6 +46,9 @@ export function MemberApprovalRequiredToggle({ memberApprovalRequired, onToggleC
         }
     }
 
+    const isDisabled = isLoading || forceMemberApprovalRequired !== undefined;
```

Comment:
similar to `forceEnableAnonymousAccess` we only need to disable if the `forceMemberApprovalRequired` flag is true